frontend/src/App.jsx

```diff
@@ -45,7 +46,9 @@ function App() {
           <Route path="/user/:id/update/recipe" element={<UpdateRecipe />} />
           <Route path="/recipe/:id" element={<OneRecipe/>} />
         </Routes>
-        <Footer />
+        {/* <Footer /> */}
+        <Footer1 />
```

Comment:
Instead of making another footer file why don't you add the new footer code to the old file only?
Please do this. And Remove Footer1.jsx
Try to add your code to existing files only unless it does not need a new file for a component or page